scripts/parseJSON.js

```diff
@@ -4,46 +4,49 @@
 const traverseObject = (obj, callback) => {
   // Array
   if (Array.isArray(obj)) {
-    return obj.map((child) => traverseObject(child, callback));
+    return obj.map(child => traverseObject(child, callback))
   }
 
   // Non-object
-  if (!obj || typeof obj !== "object") {
-    return obj;
+  if (!obj || typeof obj !== 'object') {
+    return obj
   }
 
   for (let child of Object.values(obj)) {
-    traverseObject(child, callback);
+    traverseObject(child, callback)
   }
 
-  callback(obj);
-};
+  callback(obj)
+}
 
-const [, , ...args] = process.argv;
+const [, , ...args] = process.argv
 if (args.length !== 2) {
   console.error(
-    "Usage:\n\n node install_android.js [PROJECT_PATH] [PLATFORM_ID]\n"
-  );
-  process.exit(1);
+    'Usage:\n\n node install_android.js [PROJECT_PATH] [PLATFORM_ID]\n'
+  )
+  process.exit(1)
 }
-let project_path = args[0];
-let platform_id = args[1];
-let fs = require("fs");
-let rawData = fs.readFileSync(project_path + "/proton-bundle.json");
-let protonBundle = JSON.parse(rawData);
-let appID;
-appID =
-  protonBundle.libraryGlobals["@adalo/ads"]["Ads"][`${platform_id}Global`];
+let project_path = args[0]
+let platform_id = args[1]
+let fs = require('fs')
+let rawData = fs.readFileSync(project_path + '/proton-bundle.json')
+let protonBundle = JSON.parse(rawData)
+let appID
+appID = protonBundle.libraryGlobals['@adalo/ads']['Ads'][`${platform_id}Global`]
 
 if (appID) {
-  let appIDFinal = appID.replace(/\s/g, "");
-  console.log(appIDFinal);
+  let appIDFinal = appID.replace(/\s/g, '')
+  console.log(appIDFinal)
```

Comment:
I'm not sure we need to log this out either.

Reply:
See my comment above